.flake8

```diff
@@ -9,3 +9,10 @@
 #  E731: do not assign a lambda expression, use a def
 #  E501: Line too long (black enforces this for us)
 ignore=W503,W504,E203,E731,E501
+
+# Default + Enabled flake8 error codes:
```

Comment:
I don't really understand what this sentence is trying to say. Can you rephrase?